doc/getting_started/index.rst

```diff
@@ -96,6 +96,17 @@ Clone all of Zephyr's repositories in a new :file:`zephyrproject` directory:
 You can replace :file:`zephyrproject` with another directory name. West creates
 the directory if it doesn't exist. See :ref:`west-multi-repo` for more details.
 
+.. important::
```

Comment:
I do not think this belongs into the GSG. You already ran this above, so chances someone would hit this issue when following the GSG is low. The messages below should be in the main developer guide or whatever we call it.

Reply:
We see this same question on Slack and the mailing list fairly often, so I disagree.

I think this is important enough to include here, even though I know we are trying to minimize the number of "important", "warning", etc. directives in this file.

Reply:
I think this could be part of a FAQ, but until we have that I agree that we need this somewhere visible.